train.py

```diff
@@ -123,8 +125,7 @@ def main():
             print("=> no checkpoint found at '{}'".format(opt.resume))
 
     # Train the Model
-    best_rsum = 0
-    for epoch in range(opt.num_epochs):
+    for epoch in range(start_epoch, opt.num_epochs):
```

Comment:
Wondering if you should change
`start_epoch = checkpoint['epoch']`
to 
`start_epoch = checkpoint['epoch'] + 1`

Reply:
when saving model, epoch will add 1,
`'epoch':` epoch + 1,`

thus I think it is not neccessary to change
`start_epoch = checkpoint['epoch'] + 1`

Reply:
I see, but the checkpoint filename is still 'checkpoint_{}.pth.tar'.format(epoch)'. I didn't notice this when I wrote it, but now I think it might cause confusion. Would it be better to just move '+1' to start_epoch?

Reply:
I got what you mean. Now I move '+1' to start_epoch